net/jitsi-meet-electron/Portfile

```diff
@@ -0,0 +1,41 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+
+PortSystem          1.0
+PortGroup           github 1.0
+
+github.setup        jitsi jitsi-meet-electron 2023.9.0 v
+github.tarball_from archive
+revision            0
+
+categories          net
+installs_libs       no
+license             Apache-2
+maintainers         {takikawa @takikawa} openmaintainer
```

Comment:
the first entry should be an obfuscated e-mail address. The way you have specified it now implies an "@macports.org" address which you don't have.

Reply:
Thanks, should be fixed now.